src/media/js/site/containers/login.js

```diff
@@ -4,9 +4,10 @@ import {bindActionCreators} from 'redux';
 
 import {fxaLoginBegin, login} from '../actions/login';
 import {LoginButton} from '../components/login';
+import {Page} from '../components/page';
 
 
-export class Login extends React.Component {
+export class LoginForm extends React.Component {
```

Comment:
yeah could you? Components and actions don't collide in namespace, it's login vs Login, and we have lots of Components + Actions + Reducers that match each other's names